tests/src/test/scala/com/nvidia/spark/rapids/GpuDeviceManagerSuite.scala

```diff
@@ -48,12 +47,14 @@ class GpuDeviceManagerSuite extends FunSuite with Arm with BeforeAndAfter {
         .set(RapidsConf.RMM_ALLOC_MAX_FRACTION.key, maxPoolFraction.toString)
         .set(RapidsConf.RMM_ALLOC_RESERVE.key, "0")
     TestUtils.withGpuSparkSession(conf) { _ =>
+      val freeGpuSize = Cuda.memGetInfo().free
       val poolSize = (freeGpuSize * poolFraction).toLong
+      val allocSize = Math.max(poolSize - 1024 * 1024, 0)
```

Comment:
It's nice that this protects against allocating a negative number, but do we really expect the test to fail if we try to allocate zero bytes? Maybe this should try to allocate a percentage (e.g.: ~75%) of the calculated pool size rather than a hardcoded offset, asserting the allocSize > 0.

Reply:
Done.